heartbeat/server.js

```diff
@@ -26,15 +26,16 @@ db.on('compaction.done', () => {
 
 const FLIC_THRESHOLD_MILLIS = 180*1000
 const HEARTBEAT_THRESHOLD_MILLIS = 60*1000
+const PING_THRESHOLD_MILLIS = 320*1000
 
 function log(logString) {
     console.log(moment().toISOString() + " - " + logString)
 }
 
-function sendAlerts(systemName) {
+function sendAlerts(connectionType, systemName) {
     for(let i=0; i<config.TWILIO_TO_NUMBERS.length; i++) {
         twilioClient.messages.create({
-            body: `The Flic connection for ${systemName} has been lost.`,
+            body: `The ${connectionType} connection for ${systemName} has been lost.`,
```

Comment:
Is it useful to know which threshold triggered the heartbeat alert?

Reply:
I think so! especially if the response to this is a human hopping on the command line - and starting to debug

Reply:
Yes! This is great.

Reply:
It occurs to me that `connectionType` is not really an accurate description of what the various metrics are. Similarly, something like `The Darkstat connection for BraveOffice has been lost.` is a bit misleading - what we actually want to say is more like `According to Darkstat data, the connection to BraveOffice has been lost.`

Reply:
Because in this example there is probably nothing wrong with Darkstat - the problem is actually most likely with the Flic hub.

Reply:
Good point. I've updated all the alert texts to make them more accurate. What do you about them now @schwarrrtz ?

Reply:
They all look good, except for the one typo I noted in another comment.

Reply:
Thanks! All fixed.